README.md

```diff
@@ -206,7 +213,7 @@ Now, we'll create the boundary for the end date, and run the query.
 		$adder = $month ? '+1 month' : '+1 year';
 		$endDate = date('Y-m-d', strtotime(
 		    $adder, 
-				strtotime($startDate)
+            strtotime($startDate)
```

Comment:
I'm not sure that PHP date formats work in SS4 since we've introduced CLDR date formats

Reply:
Being more a review and 'editing suggestion' type of a PR, I've assumed that the code has been tested by @unclecheese